TemporalConvolution.lua

```diff
@@ -13,7 +13,6 @@ function TemporalConvolution:__init(inputFrameSize, outputFrameSize,
     local nInputPlane = 1 -- single channel
     local nOutputPlane = outputFrameSize
     self.inputFrameSize = inputFrameSize
-    self.outputFrameSize = outputFramesize
```

Comment:
Good catch with typo. No, it is not used in this file, but if someone is using nn.TemporalConvolution, uses its outputFrameSize somewhere, and then replaces it with cudnn.TemporalConvolution that does not set it, it will break.